.jenkins

```diff
@@ -134,7 +134,9 @@ cmake \
                       git config user.email "noreply@ornl.gov" &&
                       git config user.name "Jenkins CI" &&
                       git commit -m "Automated documentation update" &&
-                      git push https://$GIT_CREDS@github.com/ORNL/TASMANIAN.git gh-pages'''
+                      git reset --soft HEAD^^ &&
```

Comment:
Why 2x ^?

Reply:
If I understand the logic:
* move head to the last commit that should not be changed
* take everything after that commit and merge it into a single commit
Using only one ^ will redo the last commit, i.e., no change at all.

Reply:
I've tested it.